py/desispec/qa/qalib.py

```diff
@@ -301,8 +301,7 @@ def SN_ratio(flux,ivar):
     medsnr=np.zeros(flux.shape[0])
     #totsnr=np.zeros(flux.shape[0])
     for ii in range(flux.shape[0]):
-        signalmask=flux[ii,:]>0 #- mask negative values
-        snr=flux[ii,signalmask]*np.sqrt(ivar[ii,signalmask])
+        snr=flux[ii]*np.sqrt(ivar[ii])
         medsnr[ii]=np.median(snr)
```

Comment:
I'd like to check with Kyle and David about what we do for BOSS SoS (quicklook) in the presence of noisy negative signal when calculating S/N .  If you have other info that motivates this choice, please comment.